Common subdirectories: mutt-1.5.20/contrib and mutt-1.5.20-2/contrib
Common subdirectories: mutt-1.5.20/doc and mutt-1.5.20-2/doc
Common subdirectories: mutt-1.5.20/imap and mutt-1.5.20-2/imap
Common subdirectories: mutt-1.5.20/intl and mutt-1.5.20-2/intl
Common subdirectories: mutt-1.5.20/m4 and mutt-1.5.20-2/m4
diff -u mutt-1.5.20/newsrc.c mutt-1.5.20-2/newsrc.c
--- mutt-1.5.20/newsrc.c	2010-07-11 14:47:48.568876588 -0400
+++ mutt-1.5.20-2/newsrc.c	2010-07-11 14:48:22.227622656 -0400
@@ -193,9 +193,9 @@
   strfcpy (dir, NewsCacheDir, sizeof (dir));
   mutt_expand_path (dir, sizeof(dir));
 
-  if (lstat (dir, &st) || (st.st_mode & S_IFDIR) == 0)
+  if (stat (dir, &st) || (st.st_mode & S_IFDIR) == 0)
   {
-    snprintf (buf, sizeof(buf), _("Directory %s not exist. Create it?"), dir);
+    snprintf (buf, sizeof(buf), _("Directory %s does not exist. Create it?"), dir);
     if (mutt_yesorno (buf, M_YES) != M_YES || mkdir (dir, (S_IRWXU+S_IRWXG+
 	  S_IRWXO)))
     {
Common subdirectories: mutt-1.5.20/po and mutt-1.5.20-2/po
